### PR TITLE
[MOSIP-31911] Automated Creation of readuser for postgres,minio and k…

### PR DESCRIPTION
**Pull request by Mohanraj209**
…eycloak service.

### CODE REVIEW

File: deployment/v3/utils/readuser-util/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# ReadUser Creation Utility:
+
+This Utility is used for the creation of user with readonly privilages for postgres, minio and keycloak server.
+
+This utility is designed to initialize MinIO with user management and policy attachment based on the specified action (create or delete) and to run a script that creates a read-only user in a PostgreSQL database and also to initialize keycloak with user with readonly privilages.
```
Comment (ckm007) [inline deployment/v3/utils/readuser-util/README.md:5]: add in points

File: deployment/v3/utils/readuser-util/install.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+CHART_VERSION=0.0.1-develop
+
+echo Create $NS namespace
+kubectl create ns $NS
+
+function installing_readuser-util() {
+
+  echo Installing readuser-util
+  helm -n $NS install readuser-util mosip/readuser-utill --version $CHART_VERSION -f values.yaml --wait
+  helm -n $IAM_NS install readuser-iam-init mosip/keycloak-init --version $CHART_VERSION -f readuser-init-values.yaml --wait
```
Comment (ckm007) [inline deployment/v3/utils/readuser-util/install.sh:20]: run keycloak init for readuser from same readuser-util namespace.

File: deployment/v3/utils/readuser-util/README.md (hunk 1 above)
Comment (Mohanraj209) [reply deployment/v3/utils/readuser-util/README.md:5]: Updated

File: deployment/v3/utils/readuser-util/install.sh (hunk 2 above)
Comment (Mohanraj209) [reply deployment/v3/utils/readuser-util/install.sh:20]: Updated

**Review by ckm007 — APPROVED** [review]

**Pull request merged**